### PR TITLE
remove circleci doc

### PR DESCRIPTION
**Pull request by aclevername**
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by nikimanoledaki** [comment]
I dont approve :trollface:

**Review by nikimanoledaki — APPROVED** [review]
ok I approve

**Review by Skarlso — APPROVED** [review]
<3 :D

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -482,7 +482,7 @@ git push <remote-name> <feature-name>
 
 If your PR is still a work in progress, please open a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 
-Our CircleCI integration will run the automated tests and give you feedback in the review section. We will review your
+Our GitHub actions integration will run the automated tests and give you feedback in the review section. We will review your
```
Comment (nikimanoledaki) [inline CONTRIBUTING.md:485]:
```suggestion
Our GitHub Actions integration will run the automated tests and give you feedback in the review section. We will review your
```

Comment (nikimanoledaki) [reply CONTRIBUTING.md:485]: :trollface:

**Pull request merged**